camelot/plotting.py

```diff
@@ -37,21 +71,23 @@ def plot_table(table):
     table : camelot.core.Table
 
     """
+    fig = plt.figure()
+    ax = fig.add_subplot(111, aspect='equal')
     for row in table.cells:
         for cell in row:
             if cell.left:
-                plt.plot([cell.lb[0], cell.lt[0]],
+                ax.plot([cell.lb[0], cell.lt[0]],
                          [cell.lb[1], cell.lt[1]])
             if cell.right:
-                plt.plot([cell.rb[0], cell.rt[0]],
+                ax.plot([cell.rb[0], cell.rt[0]],
                          [cell.rb[1], cell.rt[1]])
             if cell.top:
-                plt.plot([cell.lt[0], cell.rt[0]],
+                ax.plot([cell.lt[0], cell.rt[0]],
                          [cell.lt[1], cell.rt[1]])
             if cell.bottom:
-                plt.plot([cell.lb[0], cell.rb[0]],
+                ax.plot([cell.lb[0], cell.rb[0]],
                          [cell.lb[1], cell.rb[1]])
-    plt.show()
+    return fig
```

Comment:
@Suyash458 Are you returning fig from the plot_* functions only for those additional asserts in the tests?

Reply:
[pytest-mpl's](https://github.com/matplotlib/pytest-mpl) image comparison decorator requires the plot functions to return a matplotlib figure